.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
```

Comment:
I suggest removing the indentation inside the curly braces `{}` by analogy with the square brackets `[]`.

```suggestion
    "trailingComma": "all",
    "bracketSpacing": false,
```

Reply:
Uhm, I'd prefer spaced `{}` :)